components/webpack/webpack.config.js

```diff
@@ -116,6 +116,10 @@ module.exports = async function (env, argv) {
             return callback(null, 'module ' + request);
           }
         }
+        if (/^chrome-untrusted:\/\/resources\/brave\/candle_embedding_gemma/
+            .test(request)) {
+          return callback(null, 'module ' + request);
+        }
```

Comment:
It might be cleaner if this lived inside the `if (env.output_module)` block above.